corehq/reports.py

```diff
@@ -89,12 +88,13 @@ def REPORTS(project):
 
     if project.commtrack_enabled:
         reports.insert(0, (ugettext_lazy("CommCare Supply"), (
-            commtrack_reports.SimplifiedInventoryReport,
-            commtrack_reports.InventoryReport,
-            commtrack_reports.CurrentStockStatusReport,
-            commtrack_maps.StockStatusMapReport,
-            commtrack_reports.ReportingRatesReport,
-            commtrack_maps.ReportingStatusMapReport,
+            commtrack.SimplifiedInventoryReport,
+            commtrack.InventoryReport,
+            commtrack.CurrentStockStatusReport,
+            commtrack.StockStatusMapReport,
+            commtrack.ReportingRatesReport,
+            commtrack.ReportingStatusMapReport,
+            commtrack.LedgersByLocationReport,
```

Comment:
see my comment above - I think you'll have to check for the feature flag here and only add the report to this list if it's enabled